lib/web/apiserver.go

```diff
@@ -3138,17 +3150,7 @@ func (h *Handler) createSSHCert(w http.ResponseWriter, r *http.Request, p httpro
 			Token:    req.OTPToken,
 		}
 	case constants.SecondFactorWebauthn:
-		// WebAuthn only supports this endpoint for headless login.
-		authReq.HeadlessAuthenticationID = req.HeadlessAuthenticationID
-
-		// create a new http client with a standard callback timeout.
-		authenticationClient, err = authClient.CloneHTTPClient(
-			auth.ClientParamTimeout(defaults.CallbackTimeout),
-			auth.ClientParamResponseHeaderTimeout(defaults.CallbackTimeout),
-		)
-		if err != nil {
-			return nil, trace.Wrap(err)
-		}
+		// WebAuthn supports this endpoint for headless login.
```

Comment:
I'm not sure I follow this comment. Rephrase?

Shouldn't this block initialize some WebAuthn-specific structure?

Reply:
Removed in favor of handling headless authentication as a special case above.